### DISCUSSION TITLE
wemos lolin d32 build

### DISCUSSION THREAD

**Discussion by heidepiek** [PlummersSoftwareLLC/NightDriverStrip#571] [Show and tell]
For anyone who would also like to have such a nice mesmerizer in their mancave, here the files for a pcb for a wemos lolin D32 pro board. Also modified dave`s housing a bit to my own taste with the mic and ir eye built in. With this configuration it now sits among my other gadgets. It is not complicated, just offer the gerber files at jlcpcb.
They make a nice pcb for you, solder a few components on the board and you're done. On the pcb it is neatly stated which pinout goes to the microphone and which to the ir sensor. 
I printed the housing with text in it. I do not have a multi color printer. So with my 3d printer I first printed the text and then the housing. Tricky, but doable.  There are no special actions needed to upload the nightdriver mesmerizer on the lolin! Pinout is the same as what dave uses with his hardware! Just use in the platformio as env: mesmerizer_lolin. Even if you use env:mesmerizer it just works!👍 

The pcb files: 
[BOM_PCB_schematic lolin D32 hub75 shield_2023-12-17.csv](https://github.com/PlummersSoftwareLLC/NightDriverStrip/files/13696665/BOM_PCB_schematic.lolin.D32.hub75.shield_2023-12-17.csv)
[Gerber_PCB_schematic lolin D32 hub75 shield_2023-12-17.zip](https://github.com/PlummersSoftwareLLC/NightDriverStrip/files/13696666/Gerber_PCB_schematic.lolin.D32.hub75.shield_2023-12-17.zip)
[PickAndPlace_PCB_schematic lolin D32 hub75 shield_2023-12-17.csv](https://github.com/PlummersSoftwareLLC/NightDriverStrip/files/13696667/PickAndPlace_PCB_schematic.lolin.D32.hub75.shield_2023-12-17.csv)

The housing: 
[mesmerizer housing.zip](https://github.com/PlummersSoftwareLLC/NightDriverStrip/files/13696676/mesmerizer.housing.zip)

Some pictures of how it looks like:  
![foto 1](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/1e1790be-00c3-409a-b0b1-66fe17368202)
![foto 2](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/6fe5f7b2-78a7-4c91-81ab-fb607c5133de)
![foto 3](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/db7aadda-92ef-47d1-9d5b-a203074b89fe)
![foto 4](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/24ac156d-9c68-4daa-bb61-e0fb2e32a4aa)
![foto 5](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/32dc7271-f968-4fe6-9d8a-850486b0b313)
![foto6](https://github.com/PlummersSoftwareLLC/NightDriverStrip/assets/104028471/4bf54bec-9bf2-4e6f-80a3-cce7e7f640b1)

Happy tinkering!

Greetz harrie

**Comment by smilima** [comment]
Nice job!

**Comment by LoveMyKeys** [comment]
Cool! I ordered some boards and components to build a few for me and my son.

**Reply by heidepiek** [reply]
@LoveMyKeys
How far along are you with project pcb? Am curious how it turned out. Always nice when you get feedback.  

regards Harrie

**Reply by LoveMyKeys** [reply]
Hey Harrie,

I have it up and running without any hustle. My son is developing a 3d printed case.

Thanks for your design!

Greetz, 

LMK

**Reply by heidepiek** [reply]
Nice to hear that everything is functioning properly. If your son has come up with a nice case, I'd love to hear about it

greetz
Harrie

**Comment by davepl** [comment]
Very nice!  And quite compact.. the FTDI debug chip on my Mesmerizer board takes up a lot of space!

**Reply by heidepiek** [reply]
Hey dave, is this compliment meant for my homemade pcb? 😳 It's been a while since I posted this (Dec 17, 2023) I thought the creator of nightdriver was pretty up to date on what's going on here! On the github page nightdriverstrip I couldn't find any gerber files from your pcb. So I challenged myself to make one. First time! Nicely succeeded I think. ☺️ Had the mesmerizer running with a lolin D32, so thought I'd make a handy pcb for that. Still stands showing off next to my other gadgets. Nice piece of software!🤘 

Greetz Harrie👍
